tests/controllers/TalksControllerTest.php

```diff
@@ -33,17 +44,23 @@ public function testClaimTalkWithNoUserIdThrowsException()
      *
      * @return void
      *
-     * @test
```

Comment:
Though you are absolutely right, that's a refactoring that should also go into it's own PR